src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(7 * $day-size + 6 * $gap + 2 * $padding);
```

Comment:
Ensure that the variables `$day-size`, `$gap`, and `$padding` are defined in your SCSS files. Otherwise, this line will cause a compilation error.